cl/settings/project/corpus_importer.py

```diff
@@ -0,0 +1,14 @@
+import environ
+
+env = environ.FileAwareEnv()
+IQUERY_PROBE_ITERATIONS = env.int("IQUERY_PROBE_ITERATIONS", default=9)
+IQUERY_PROBE_WAIT = env.int("IQUERY_PROBE_WAIT", default=300)
+IQUERY_PROBE_MAX_CONSECUTIVE_FAILURES = env.int(
+    "IQUERY_PROBE_MAX_CONSECUTIVE_FAILURES", default=5
+)
```

Comment:
I noticed this setting isn't referenced anywhere in the code.

Reply:
Yeah, this is unused now, removed.